alpha/declcfg/load.go

```diff
@@ -168,3 +168,20 @@ func readYAMLOrJSON(r io.Reader) (*DeclarativeConfig, error) {
 	}
 	return cfg, nil
 }
+
+func LoadFile(root fs.FS, file fs.File, path string) (*DeclarativeConfig, error) {
+	var (
+		cfg = &DeclarativeConfig{}
+		err error
+	)
+
+	if cfg, err = readYAMLOrJSON(file); err != nil {
+		return nil, err
+	}
```

Comment:
Thoughts?
```suggestion
	if cfg, err := readYAMLOrJSON(file); err != nil {
		return nil, err
	}
```

Reply:
I tried that, but then I get errors:
```alpha/declcfg/load.go:173:5: cfg declared but not used
alpha/declcfg/load.go:177:30: undefined: cfg
alpha/declcfg/load.go:178:10: undefined: cfg
alpha/declcfg/load.go:181:9: undefined: cfg
make: *** [bin/opm] Error 2
```

Reply:
Ah, no you had it right before. Forgot that you're checking the error inline here.

Reply:
The more idiomatic way to do this is:
```suggestion
	cfg, err := readYAMLOrJSON(file)
	if err != nil {
		return nil, err
	}
```